src/ts/ssh/sshChannel.ts

```diff
@@ -271,6 +277,61 @@ export class SshChannel implements Disposable {
 		}
 	}
 
+	public async sendExtendedData(dataTypeCode: SshExtendedDataType, data: Buffer, cancellation?: CancellationToken): Promise<void> {
```

Comment:
Would it be possible to refactor to eliminate some of the duplication of code between this method and `send()` above?

Reply:
@jasongin  Addressed, but don't know if you like `sendCommon` function or not 🙏